src/gui/owncloudgui.cpp

```diff
@@ -897,19 +898,21 @@ void ownCloudGui::runNewAccountWizard()
 
                                     folderMan->setSyncEnabled(true);
                                     folderMan->scheduleAllFolders();
+                                    accountStatePtr->setSettingUp(false);
```

Comment:
Just a question in general, would it be more elegant to use a setter like this or rather emit a signal on the account state pointer? I saw we need to track the state anyway in a local variable.

Reply:
Emitting signals for other objects feels "wrong" additionally the setter has the benefit that we can query the state and not rely only on signals.